src/wallet/wallet.ts

```diff
@@ -1099,6 +1099,57 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return parseInt(parts[5], 10);
   }
 
+
+  /**
+   * Get mint authorities
+   * Uses the getUtxos method to return one or many mint authorities
+   *
+   * @param tokenId of the token to select the authority utxo
+   * @param options Object with custom options.
+   *  {
+   *    'many': if should return many utxos or just one (default false),
+   *    'skipSpent': if should not include spent utxos (default true)
+   *  }
+   *
+   * @return Array of objects with {tx_id, index, address} of the authority output. Returns null in case there are no utxos for this type
+   **/
+  async getMintAuthority(tokenId: string, options: { many?: boolean, skipSpent?: boolean } = {}) {
+    const newOptions = Object.assign({ many: false, skipSpent: true }, options);
+    const { utxos } = await this.getUtxos({
+      tokenId,
+      authority: TOKEN_MINT_MASK,
+      skipSpent: newOptions.skipSpent,
+      count: newOptions.many ? undefined : 1,
+    });
+
+    return utxos;
+  }
+
+  /**
+   * Get melt authorities
+   * Uses the getUtxos method to return one or many mint authorities
```

Comment:
melt*

Reply:
Fixed in c427d6afea611a3f335cdec5dedc57a4f6947d11